lightningd/peer_htlcs.c

```diff
@@ -940,6 +940,16 @@ static bool htlc_accepted_hook_deserialize(struct htlc_accepted_hook_payload *re
 				      " hook: %.*s",
 				      failmsgtok->end - failmsgtok->start,
 				      buffer + failmsgtok->start);
+        } else if ((failoniontok = json_get_member(buffer, toks,
```

Comment:
Nit: indentations are tabs in our project. Note: We use a visual tab-width of 8 spaces.

Reply:
In which circumstances are spaces used? I do see them in many lines mixed with the tabs.

Reply:
Ah, you can refer to `doc/STYLE.md` (Tabs and indentation=

Reply:
Basically its tabs with a visual length of 8 spaces. If you break up e.g. a function call to align the arguments each line then you can use spaces (after the tabs) to make them start at exactly the indentation you need.

Reply:
Thank you.
These are rules very hard to follow, but I like them.

Reply:
Check the `.editorconfig` in the repos basedir. My VIM editor respects it.